src/Features/LanguageServer/ProtocolUnitTests/CodeActions/CodeActionsTests.cs

```diff
@@ -168,14 +168,42 @@ class ABC
             };
 
             var results = await RunGetCodeActionsAsync(testLspServer, codeActionParams);
-            Assert.Equal(7, results.Length);
+            Assert.Single(results);
```

Comment:
This is changed to `Single` because `Suppress or configure issues` are filtered out.

I added another test to cover the original logic. (flatten the nested code actions)